Daytime-Nighttime-Interactions/announce-component-usage.rst

```diff
@@ -37,9 +37,9 @@ Overview
 
 This topic covers how to properly announce the usage of component(s) on the summit.
 
-.. _Announcing-Component-Usage-Prerequisites:
+.. _Announcing-Component-Usage-Precondition:
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```

Comment:
^^^^^ ---> ===== ?

Reply:
Changed also the rest of the sections to ========, and in other documents within Daytime-Nighttime-Interactions that were using old templates.